### ISSUE TITLE
Error in executing the binary executable

### ISSUE THREAD

**Issue by PritAnmol** [godotengine/godot#16721]
**Godot version: Current master branch**
Commit Hash :- 133942cfeb0279ca03ab9943459c5140ab1f1725

**OS/device including version:  Windows 10 (OS Build 16299.248)**
GPU Model :  GeForce 940MX/PCIe/SSE2

**Issue description:**
I followed each and every step in the documentation for "Compiling in Windows."
I compiled the source code on windows 10 using Scons.
The executable was successfully created inside the bin directory as directed in the documentation.
But when I ran the executable nothing shows up apart from a white screen
![godotsetup](https://user-images.githubusercontent.com/22765683/36268325-2c533008-129c-11e8-83a7-abbf0d466f3a.png)

Error generated : ` OpenGL ES 3.0 Renderer: GeForce 940MX/PCIe/SSE2 WARNING: StreamPeerMbedTLS::_load_certs: Error parsing some certificates: 1      At: modules\mbedtls\stream_peer_mbed_tls.cpp:281 `

**Steps to reproduce:**
I cloned the source code using GIT.
Then I compiled using `scons -j3 platform=windows` given I have 4 cores of processor.
Then I executed the .exe file give generated the error specified above.

**Comment by AlexHolly** [comment]
I get the same error
Master Kubuntu 
![screenshot from 2018-02-16 01-57-43](https://user-images.githubusercontent.com/4741886/36288916-6ce64468-12bd-11e8-9b2a-fb666012a527.png)

**Comment by akien-mga** [comment]
@AlexHolly This bug report is about Godot not displaying anything. The error you're mentioning is most likely unrelated, so please report it in its own issue.

**Comment by virtualpain** [comment]
I found it strange, I have exact same GPU Model, and it is indeed shows blank white screen on Windows (I have yet to test it on Ubuntu in same machine) to all version of Godot including 2.x too. I tried this on many machines, Godot runs in all of them except mine. Is this more hardware/driver specific problem?
![ss1](https://user-images.githubusercontent.com/3953905/37469833-c39bb126-2898-11e8-940e-8386f7f82aaa.png)
